pkg/kubectl/cmd/util/shortcut_restmapper.go

```diff
@@ -40,20 +40,22 @@ func NewShortcutExpander(delegate meta.RESTMapper, client discovery.DiscoveryInt
 	return ShortcutExpander{All: UserResources, RESTMapper: delegate, discoveryClient: client}
 }
 
-func (e ShortcutExpander) getAll() []schema.GroupResource {
+func (e ShortcutExpander) getAll() ([]schema.GroupResource, []schema.GroupResource) {
```

Comment:
hm, maybe it'd be better to check if any of the hardcoded `UserResources` matched any of the  available server resources here and if the final list is empty, return `availableResources` as the only slice?